h2o-py/tests/testdir_jira/pyunit_pubdev_4702_algo_max_runtime_secs_large.py

```diff
@@ -68,32 +68,32 @@ def algo_max_runtime_secs():
     print("******************** Skip testing Naives Bayes.  Not an iterative algo.")
 
     # random foreset
-    model = H2ORandomForestEstimator(ntrees=100, score_tree_interval=0)
+    model = H2ORandomForestEstimator(ntrees=100, score_tree_interval=0, seed=seed)
     grabRuntimeInfo(err_bound, 2.0, model, training1_data, x_indices)
     cleanUp([model, training1_data])
 
-    # deepwater
-    if H2ODeepWaterEstimator.available():
-        training1_data = h2o.import_file(path=pyunit_utils.locate("smalldata/gbm_test/ecology_model.csv"))
-        training1_data = training1_data.drop('Site')
-        training1_data['Angaus'] = training1_data['Angaus'].asfactor()
-        y_index = "Angaus"
-        x_indices = list(range(1, training1_data.ncol))
-        model = H2ODeepWaterEstimator(epochs=50, hidden=[4096, 4096, 4096], hidden_dropout_ratios=[0.2, 0.2, 0.2])
-        grabRuntimeInfo(err_bound, 2.0, model, training1_data, x_indices, y_index)
-        cleanUp([training1_data, model])
+    # # deepwater
```

Comment:
I think we can remove these lines. :) @Pscheidl had already removed deepwater from h2o-3.

Reply:
Yes @maurever is right. This is kind-of tricky, as deepwater removal will be done in next major. So it's only in master. Everything related to DeepWater should be ignored now, as it gone.

Reply:
Let me remove it then.